main.go

```diff
@@ -163,8 +165,32 @@ func mailHandler(peer smtpd.Peer, env smtpd.Envelope) error {
 		"uuid": generateUUID(),
 	})
 
+	if *remoteHost == "" && *command == "" {
+		logger.Warning("no remote_host or command set; discarding mail")
+		return nil
+	}
+
+	if *command != "" {
+		cmdLogger := logger.WithField("command", *command)
+
+		var stdout bytes.Buffer
+		var stderr bytes.Buffer
+
+		cmd := exec.Command(*command)
+		cmd.Stdin = bytes.NewReader(env.Data)
```

Comment:
Okay so this is only able to pass the mail to a script but the script is unable to modify and return it.

My thought was that modify and return would allow features like spamchecks, signing etc.
Right now this allows featues like backup, send via another channel etc.

Reply:
It's okay for me, just wanted to point out that this does only allow to pass mails but not modify/return them in any way.